### ISSUE TITLE
{Linked:Bug39040012}[Agent Builder Skills] [Web][Testing an agent with a skill in the Preview tab] [ScreenReader]: Copied message is disappearing immediately and users are unable to get that message

### ISSUE THREAD

**Issue by Kjasnani7** [microsoft/BotFramework-WebChat#5857]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Users may not know whether the copy action was successful because the confirmation message disappears before it can be read or perceived. Screen reader users, low-vision users, and users with cognitive disabilities may miss the status update, leading to confusion and repeated actions.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

ARIA roles

### What is the public URL for the website?

https://aka.ms/agentbuilder-native-msit

### How to reproduce the issue?

1. Launch the application using the URL: https://aka.ms/agentbuilder-native-msit and login with appropriate credentials. TAB to "Agents ans skills" and press ENTER. Select add add and press ENTER on skip.
2. Create an agent with a valid skill.
3. TAB to the "Preview" tab and press ENTER.
4. Enter a prompt that invokes the added skill and submit it.
5. Observe the issue

### What do you expect?

After the copy action is performed, the confirmation message should remain visible long enough to be perceived and should be announced to assistive technologies through an appropriate status message mechanism

### What actually happened?

Copied message is disappearing immediately and users are unable to get that message

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/728cc1c8-89d3-4972-9eec-1807184a56c7

### MAS reference

https://aka.ms/MAS4.1.3

### Adaptive Card JSON

```json

```

### Additional context

https://github.com/user-attachments/assets/8072b1f5-bd45-43fd-9ba1-9f68b84da068
